wire-tap/wire-tap.bal

```diff
@@ -0,0 +1,44 @@
+import ballerina/http;
+
+type SnowflakeRequest record {
+    string statement;
+    string timeout = "60";
+    string database = "messageLog";
+    string schema = "message";
+    string role = "logger";
+};
+
+type WireTapRequest record {|
+    string 'table;
+    "INFO"|"WARNING"|"ERROR" severity;
+    string message;
+|};
+
+type StockResponse record {
```

Comment:
better to use closed records.

Reply:
Response may have other non-required fields